### ISSUE TITLE
CamelCase cache tablename

### ISSUE THREAD

**Issue by jokumer** [jweiland-net/maps2#7]
Extension adds 'maps2_cachedHtml' to cacheConfigurations, which creates tablenames with a uppercase char. Switching the system case sensitivity and mysql lower_case_table_names settings can break use of tables 'cf_maps2_cachedHtml' and 'cf_maps2_cachedHtml_tags'.
To avoid such use case, simply use lowercase chars only in cache configuration name.

**Comment by froemken** [comment]
Good point. Thank you. I just have updated maps2 to version 2.5.0. Yeah...this change is breaking. I have added some update instructions to documentation.

**Comment by jokumer** [comment]
Hello Stefan,
I think, I did reach a very very rare case. I hope, the change will not break running installations.

I just switched my OSX environment from default caseinsesitivity to casesensitiv, and got a lot new errors, not seen before. Yeah...
To solve cf_maps2_cachedHtml tables, i should rename db-files to lower, delete them manually and install via Em again, to got EXT:map2 running again.

best regards, Jörg
